libs/grok/src/main/java/org/elasticsearch/grok/Grok.java

```diff
@@ -167,7 +174,7 @@ public String groupMatch(String name, Region region, String pattern) {
      *
      * @return named regex expression
      */
-    public String toRegex(String grokPattern) {
+    protected String toRegex(String grokPattern) {
```

Comment:
This *feels* private, but it is used by tests. I tightened the visibility while I was there, but not didn't refactor a bunch of tests.